WalletLibrary/WalletLibrary/Requests/Requirements/VerifiedIdRequirement.swift

```diff
@@ -23,6 +23,13 @@ public class VerifiedIdRequirement: Requirement {
     /// An optional property for information needed for issuance during presentation flow.
     public let issuanceOptions: [VerifiedIdRequestInput]
     
+    /// The verified id that fulfilled this requirement (if one exists)
+    public var verifiedId: VerifiedId? {
```

Comment:
Will a developer be able to set this value?

Reply:
Tested and you get a compiler error "Cannot assign to property: 'verifiedId' is a get-only property"